tools/fixedpointarithmetic/src/main/java/dk/alexandra/fresco/fixedpoint/SFixed.java

```diff
@@ -0,0 +1,21 @@
+package dk.alexandra.fresco.fixedpoint;
+
+import dk.alexandra.fresco.framework.DRes;
+import dk.alexandra.fresco.framework.value.SInt;
+
+/**
+ * Wrapper for SInt used to represent fixed point values.
+ */
+public class SFixed {
```

Comment:
Similarly to the SInt this should be an interface and then there should be a single implementation called SFixedSIntWrapper